great-idea/index.html

```diff
@@ -15,9 +15,74 @@
 </head>
 
   <body>
+    <div class="container home-page">
+      <nav>
+        <div class="navcontainer">
+          <div class="navmenu">
+            <a href="services.html" class="menu_services">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </div>
+          <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+        </div>
+      </nav>
+      <header class="headermain">
+        <div class="headertext">
+          <h1>Innovation<br>
+          On
+          <br>
+          Demand</h1>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+          <p>Get Started</p>
+        </div>
+
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </header>
+    <hr>
+      <section class="above-img">
+        <div class="features">
+          <h3>Features</h3>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+
+        <div class="about">
+          <h3>About</h3>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+      </section>
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+      <section class="below-img">
+        <div class="services">
+          <h3>Services</h3>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+      <div class="product">
+          <h3>Product</h3>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+        <div class="vision">   
+          <h3 class="vision">Vision</h3>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div> 
+      </section>
+    <hr>
+      <address>
+        <h2>Contact</h2>
+            <p>123 Way 456 Street<br>
+            Somewhere, USA<br><br>
+            1 (888) 888-8888<br><br>
+            sales@greatidea.io</p>
+      </address>
+      
+      <footer>
+        <p>Copyright Great Idea! 2018</p>
+      </footer>
       
```

Comment:
You did a great job! You're clearly an advanced user for html/css. I'd love to see you push yourself a little more with the stretch! Your code is clean and mostly easy to read.